bundle/phases/destroy.go

```diff
@@ -118,6 +120,7 @@ func Destroy(ctx context.Context, b *bundle.Bundle) (diags diag.Diagnostics) {
 	diags = diags.Extend(bundle.ApplySeq(ctx, b,
 		terraform.StatePull(),
 		terraform.Interpolate(),
+		resourcemutator.ConfigureDashboardSerializedDashboard(),
```

Comment:
Adding this mutator to this and the other phases seems like an anti-pattern. Isn't there one place it can be added such that it is valid for all cases where it needs to be valid?

Reply:
I put them here in order to be as close as possible to the original conversion of the file_path field (which happened on `terraform.write()`). I will try to find another spot in the pipeline to fit this in.

Reply:
as discussed, we place this mutator as part of the PythonMutator